advanced/Scripts/piholeCheckout.sh

```diff
@@ -137,6 +142,8 @@ checkout() {
             exit 1
         fi
         checkout_pull_branch "${PI_HOLE_FILES_DIR}" "${2}"
+        # Force an update of the updatechecker
+        updateCheckFunc
```

Comment:
Not needed as it is already run during the install script (which is run when core branches are checked out)

Reply:
removed